reference-architecture/aws-ansible/playbooks/openshift-setup.yaml

```diff
@@ -21,20 +21,23 @@
     osm_cluster_network_cidr: 172.16.0.0/16
     openshift_registry_selector: "role=infra"
     openshift_router_selector: "role=infra"
+    openshift_hosted_router_replicas: 2
+    openshift_hosted_registry_replicas: 2
     openshift_master_cluster_method: native
     openshift_cloudprovider_kind: aws
     openshift_master_cluster_hostname: "internal-openshift-master.{{ public_hosted_zone }}"
     openshift_master_cluster_public_hostname: "openshift-master.{{ public_hosted_zone }}"
     osm_default_subdomain: "{{ wildcard_zone }}"
+    osm_default_node_selector: "role=app"
     deployment_type: openshift-enterprise
     openshift_master_identity_providers:
     - name: github
       kind: GitHubIdentityProvider
       login: true
       challenge: false
       mapping_method: claim
-      clientID: e72863337b0417387b53
-      clientSecret: a2230462295d6b579d25f46dd51eb25a170e7e60
+      clientID: e76865557b0417387b35 
```

Comment:
I just noticed that clientID and clientSecret are here. Doesn't seem like something that should be in repo.